### DISCUSSION TITLE
Potential significant performance gains with NebulyAI Speedster

### DISCUSSION THREAD

**Discussion by tripletto** [vladmandic/sdnext#938] [Ideas]
Hi @vladmandic,
I don't know if you have this already on your radar or if it's even applicable and / or feasible to implement but I just came across their GitHub page and thought I just leave this here – just in case:

https://github.com/nebuly-ai/nebuly/tree/main/optimization/speedster

**Comment by vladmandic** [comment]
interesting, but somewhat intrusive - it does a lot with little control.
